Components/TodoList.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { StyleSheet, View, Text } from 'react-native'
+import Icon from 'react-native-vector-icons/Feather'
+import styled from 'styled-components'
+
+export default function TodoList(props) {
+  return (
+    <ListContainer>
+      <Icon
+        name={props.checked ? 'check' : 'square'}
```

Comment:
nice little ternary here, to show the correct icon.
This is a good use of props, for example.